src/color/ui.json

```diff
@@ -51,7 +51,7 @@
       },
       "disabled": {
         "onLight": {
-          "value": "{color.base.ui-opacity-20.value}",
+          "value": "{color.brand.atlas.400-opacity-20.value}",
```

Comment:
Need to potentially change this based on if we make any changes to the disabled states.

Reply:
ok, let me know what you want to do.